end-to-end-test/shared/wdio/webdriver-manager.conf.js

```diff
@@ -319,21 +333,17 @@ var config = {
     // }
 };
 
-const doBrowserstack = false;
-
-if (doBrowserstack) {
-    config.capabilities[0]['browserstack.local'] = true;
+if (process.env.TEST_IE11 === 'true') {
+    config.capabilities = config.IECapabilties;
+}
 
+if (process.env.TEST_BROWSERSTACK === 'true') {
     config.services = ['visual-regression', 'browserstack'];
-
     config.browserstackLocal = true;
-
-    config.user = process.env.BROWSERSTACK_USER;
-    config.key = process.env.BROWSERSTACK_KEY;
+    config.user = 'aaronlisman2'; //process.env.BROWSERSTACK_USER;
+    config.key = 'SsvxMVGa2fFVc5bWzswx'; // process.env.BROWSERSTACK_KEY;
```

Comment:
maybe good to put these in cirlceci env config instead i.e. not in the github repo? Ofc in practice somebody could still steal the keys by opening a PR and echo'ing these vars but it's a bit more obfuscated